frontend/src/component/releases/ReleasePlanTemplate/TemplateForm/TemplateForm.tsx

```diff
@@ -6,7 +6,6 @@ import { TemplateFormDescription } from './TemplateFormDescription';
 import { MilestoneList } from './MilestoneList/MilestoneList';
 
 const StyledInput = styled(Input)(({ theme }) => ({
-    width: '100%',
```

Comment:
Is this one intentional?

Reply:
Yeah there's a maxwidth of 50% just below, setting width > maxwidth didn't seem like a good idea?